WORKSPACE

```diff
@@ -3,7 +3,8 @@ workspace(name = "envoy")
 load("//bazel:repositories.bzl", "envoy_dependencies")
 load("//bazel:cc_configure.bzl", "cc_configure")
 
-envoy_dependencies()
+PPC_SKIP_TARGETS=["luajit",]
```

Comment:
I think it would be better to just inline this in `repositories.bzl` a a top-of-file constant; it probably won't change that much and should eventually shrink to zero. This simplifies the `WORKSPACE` by avoiding special casing.